LayoutTests/crypto/subtle/derive-key-for-all-supported-algorithms.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
+<script src="../resources/common.js"></script>
+</head>
+<body>
+<p id="description"></p>
+<div id="console"></div>
+
+<script type="text/javascript">
+description("We test deriveKey and then make sure that ecdh produces the same shared secret and the same encryption results using a key derived from that secret.");
+
+/* generating data
```

Comment:
Not needed anymore or is this for documentation purposes? If it's for documenting how you got the values, then I'd suggest adding a few more words.

Reply:
added